appcues/src/main/java/com/appcues/debugger/screencapture/Capture.kt

```diff
@@ -21,6 +23,11 @@ internal data class Capture(
     val metadata: Metadata,
     val timestamp: Date,
 ) {
+    // this is a lateinit var instead of a constructor arg so we can ignore it in JSON
+    // serialization and a default value is not needed for the Moshi generated adapter
+    @Json(ignore = true)
+    lateinit var screenshot: Bitmap
```

Comment:
later, this will be used to upload the image data, get back the image URL and then pass along the link in the screen API.